tests/test-fgesv.C

```diff
@@ -64,8 +64,10 @@ int main(int argc, char** argv){
 	}
 	int nbit=atoi(argv[4]); // number of times the product is performed
 	Field F(atoi(argv[1]));
-	F.init(zero,0.0);
-	F.init(one,1.0);
+	F.init(zero);
```

Comment:
replace these one and zero by F.one and F.zero where they are used (fgemm below) and remove them.